changelog.txt

```diff
@@ -15,6 +15,7 @@
 * Fix - Incorrect text when filtering subscriptions to no results.
 * Changed - Subscription products must have a recurring amount greater than $0.
 * Fix - Return correct product prices datatype in WCPay.
+* Fix - Prevent duplicate subscriptions from being created when submit button is spammed.
```

Comment:
Suggest using clearer language here - `spammed` is colloquial, casual. E.g. `Prevent duplicate subscriptions from being created when submit button is repeatedly clicked` or similar.

Reply:
Updated in c173a4c.